packages/lit-html/src/lit-html.ts

```diff
@@ -821,6 +823,10 @@ class TemplateInstance {
     this._$parent = parent;
   }
 
+  get _$isConnected() {
```

Comment:
Add a comment describing this.

Reply:
Documented once in the interface and added comments for each one pointing to it.